airflow/contrib/utils/bluecore_bq_gcs_task_handler.py

```diff
@@ -90,45 +88,45 @@ def _read(self, ti, try_number):
 
             # get info for the BQ table name
             data_project_name = configuration.get('core', 'kubernetes_bigquery_data_project')
-            start_date = ti.start_date.strftime("%Y%m%d")
-            end_date = datetime.utcnow().date().strftime("%Y%m%d")
 
             log_lines = []
-            tables = []
+            pods = []
 
             for pod in pod_output['items']:
-                pod_id = pod['metadata']['name']
-                for container in pod['spec']['containers']:
-                    container_name = string.replace(container['name'], '-', '_')
-                    bq_table_name = "{}.gke_logs.{}_*".format(data_project_name, container_name)
-                    tables.append((bq_table_name, pod_id))
+                # E.g: bt-recs-reaper-6d2f935e5af4512e-fk9tx
+                pods.append(pod['metadata']['name'])
```

Comment:
How are logs from multi-container pods going to appear in this world?

Reply:
Good call, do you have any example? 
I need to manually check it on the database and maybe request more data.